sdk/resourcemanager/azure-resourcemanager-samples/src/main/java/com/azure/resourcemanager/cdn/samples/ManageCdn.java

```diff
@@ -34,7 +34,7 @@
  * - Create CDN profile using Standard Verizon SKU with endpoints in each region of Web apps.
  * - Load some content (referenced by Web Apps) to the CDN endpoints.
  */
-public class ManageCdnProfileWithWebApp {
+public class ManageCdn {
```

Comment:
For now, you might want to change `SAMPLES.md` as well.

Reply:
It looks cdn/eventhubs .etc are not covered in `SAMPLE.md` yet. We could have separate PR to update it.